maestro-cli/src/main/java/maestro/cli/device/PickDeviceView.kt

```diff
@@ -0,0 +1,61 @@
+package maestro.cli.device
+
+object PickDeviceView {
+
+    fun showRunOnDevice(device: Device) {
+        println("Running on ${device.description}")
+    }
+
+    fun pickDeviceToStart(devices: List<Device>): Device {
```

Comment:
We should handle the situation where there are no devices available (eg. When the user has not set up any Android emulators and doesn't have XCode tools installed